book/website/project-design/pd-bias/pd-bias-identification.md

```diff
@@ -0,0 +1,89 @@
+These are useful checklists which prompt the user to consider whether there are limitations in the participants, predictors, outcome, and analysis included in the given research. 
+
+It is important to identify potential risks of bias while planning research so that bias mitigation strategies can be put in place.
+Creating a research protocol can be invaluable in ensuring that the research has been thoroughly planned.
+The protocol can even be peer-reviewed before the start of the active research, and can be published alongside the findings in a [registered report](https://book.the-turing-way.org/communication/dif-articles/reg.html#registered-reports). 
+
+
+(pd-bias-identification-sampling)=
+## Sampling
+In statistics, a *population* is any group of similar individuals that is being modelled.
+Individuals may be physical objects, such as people with a specific disease, or rivers in South America, or they may be conceptual, such as different news sources or political ideologies. 
+
+It is rarely possible to thoroughly analyse an entire population, so researchers will typically focus on a *sample*, which is a subset of the population. 
+The hope is that lessons learned about the sample will apply to the whole population, but this is not always the case. 
+
+Samples are typically intended to be uniform and random, meaning that every inidividual in the population has equal chance of being selected. 
+*Sampling bias* occurs when systematic issues cause some individuals to be more likely to be included than others. 
+This may be caused by *convenience sampling*, where potential participants are selected because they are easy to access. 
+
+
+In many studies, samples cannot be truly random as they depend upon members of the population volunteering themselves.
+This often leads to a *participation bias*, where some population subgroups are more able or willing to participate than others. 
+For example, it is very common for participation to be related to socio-economic factors, with healthy, wealthy, older people more likely to have the time and financial security required to volunteer in research. 
+It is often impossible to completely elimiate such biases, but they can be mitigated by taking measures to [enable and encourage](https://www.evalacademy.com/articles/incentives-for-participation) a wider group of the population to participate. 
+
+A sample may also differ from the population simply due to random chance - the only way to guarantee exact representation is to include every member of the population.
+*Stratified sampling* can keep certain characteristics consistent between the population and the sample - for example, if the gender proportions of a population are known then a stratified sample can be taken with these same proportions.
+This may be referred to as "proportional representation".
```

Comment:
```suggestion
This may be referred to as 'proportional representation'.
```
changing so you have the same quote marks throughout as you have a mix of ' and "